packages/core/src/plugins/web3Form.ts

```diff
@@ -0,0 +1,153 @@
+import { DomElement } from 'htmlparser2';
+import cheerio from 'cheerio';
+import has from 'lodash/has';
+import { PluginContext } from './Plugin';
+
+const md = require('../lib/markdown-it');
+
+const _ = {
+  has,
+};
+
+const CSS_FILE_NAME = 'web3FormAssets/web-3-form.css';
+
+const NAME_INPUT = '<label for="name">Name</label>\n'
+    + '<input type="text" name="name" required placeholder="John Doe">';
+const EMAIL_INPUT = '<label for="email">Email</label>\n'
+    + '<input type="email" name="email" required placeholder="johndoe@gmail.com">';
+const MESSAGE_INPUT = '<label for="message">Message</label>\n'
+    + '<textarea name="message" required></textarea>';
+const SUBMIT_BUTTON = '<button type="submit" class="badge bg-info">Submit</button>';
+const DEFAULT_HEADER = '<h2>Contact Us</h2>';
+const DEFAULT_INPUTS = `${NAME_INPUT}\n${EMAIL_INPUT}\n${MESSAGE_INPUT}\n${SUBMIT_BUTTON}`;
+
+function createMinimalForm(pluginContext: PluginContext) {
+  const $replaceNode = cheerio('<form onsubmit="submitForm(this)"></form>');
+  $replaceNode.append(`<input type="hidden" name="access_key" value=${pluginContext.accessKey}>`);
+  return $replaceNode;
+}
+
+function deleteWeb3FormAttributes(node: cheerio.Element & DomElement) {
+  if (!node.attribs) return;
+  delete node.attribs.default;
+  delete node.attribs.header;
+}
+
+function transformFormInputs(child: cheerio.Element) {
+  if (child.type !== 'tag') {
+    return;
+  }
+  const $node = cheerio(child);
+  switch (child.name.toLowerCase()) {
```

Comment:
Valid edge case, but I believe conversion to lowercase is not necessary as it's already done in [NodeProcessor](https://github.com/yucheng11122017/markbind/blob/f12518e6281aceae8fca50d9e336cb17df1cdf90/packages/core/src/html/NodeProcessor.ts#L321) before the pluginManager calls `processNode` :)